nf_core/lint/__init__.py

```diff
@@ -335,55 +340,74 @@ def format_result(test_results, table):
+                rich.panel.Panel(
+                    format_result(self.passed),
+                    title=r"[bold][✔] {} Pipeline Test{} Passed".format(len(self.passed), _s(self.passed)),
+                    title_align="left",
+                    style="green",
+                    padding=1,
+                )
+            )
 
         # Table of fixed tests
         if len(self.fixed) > 0:
-            table = Table(style="bright_blue", box=rich.box.ROUNDED)
-            table.add_column(r"[?] {} Test{} Fixed".format(len(self.fixed), _s(self.fixed)), no_wrap=True)
-            table = format_result(self.fixed, table)
-            console.print(table)
+            console.print(
+                rich.panel.Panel(
+                    format_result(self.fixed),
+                    title=r"[bold][?] {} Pipeline Test{} Fixed".format(len(self.fixed), _s(self.fixed)),
+                    title_align="left",
+                    style="bright_blue",
+                    padding=1,
+                )
+            )
 
         # Table of ignored tests
         if len(self.ignored) > 0:
-            table = Table(style="grey58", box=rich.box.ROUNDED)
-            table.add_column(r"[?] {} Test{} Ignored".format(len(self.ignored), _s(self.ignored)), no_wrap=True)
-            table = format_result(self.ignored, table)
-            console.print(table)
+            console.print(
+                rich.panel.Panel(
+                    format_result(self.ignored),
+                    title=r"[bold][?] {} Pipeline Test{} Ignored".format(len(self.ignored), _s(self.ignored)),
+                    title_align="left",
+                    style="grey58",
+                    padding=1,
+                )
+            )
 
         # Table of warning tests
         if len(self.warned) > 0:
-            table = Table(style="yellow", box=rich.box.ROUNDED)
-            table.add_column(r"[!] {} Test Warning{}".format(len(self.warned), _s(self.warned)), no_wrap=True)
-            table = format_result(self.warned, table)
-            console.print(table)
+            console.print(
+                rich.panel.Panel(
+                    format_result(self.warned),
+                    title=r"[bold][!] {} Pipeline Test Warning{}".format(len(self.warned), _s(self.warned)),
+                    title_align="left",
+                    style="yellow",
+                    padding=1,
+                )
+            )
 
         # Table of failing tests
         if len(self.failed) > 0:
-            table = Table(style="red", box=rich.box.ROUNDED)
-            table.add_column(r"[✗] {} Test{} Failed".format(len(self.failed), _s(self.failed)), no_wrap=True)
-            table = format_result(self.failed, table)
-            console.print(table)
+            console.print(
+                rich.panel.Panel(
+                    format_result(self.failed),
+                    title=r"[bold][✗] {} Pipeline Test{} Failed".format(len(self.failed), _s(self.failed)),
```

Comment:
Is fixed here, the snapshot shows "Pipeline TestsFailed"  (test in plural) but in the code is ok